configs/sites/orion/compilers.yaml

```diff
@@ -19,14 +19,15 @@ compilers:
       cxx: /apps/intel-2021.2/intel-2021.2/compiler/2021.2.0/linux/bin/intel64/icpc
       f77: /apps/intel-2021.2/intel-2021.2/compiler/2021.2.0/linux/bin/intel64/ifort
       fc: /apps/intel-2021.2/intel-2021.2/compiler/2021.2.0/linux/bin/intel64/ifort
-    flags:
-      cxxflags: '-gxx-name=/apps/gcc-10.2.0/gcc-10.2.0/bin/g++ -Wl,-rpath,/apps/gcc-10.2.0/gcc-10.2.0/lib64'
-      ldflags: '-gxx-name=/apps/gcc-10.2.0/gcc-10.2.0/bin/g++ -Wl,-rpath,/apps/gcc-10.2.0/gcc-10.2.0/lib64'
+    flags: {}
     operating_system: centos7
     target: x86_64
     modules:
     - intel/2021.2
-    environment: {}
+    environment:
+      prepend_path:
```

Comment:
Because this way CPATH makes it into the compiler meta module and other applications will be able to see the non-default gcc include files.

Reply:
I'm not familiar with CPATH

Reply:
`CPATH` specifies a list of directories to be searched as if specified with -I, but after any paths given with -I options on the command line. This environment variable is used regardless of which language is being preprocessed.

For example, the `crtm` module produced by spack has:
```
prepend_path("CPATH","/Users/heinzell/work/jedi-stack/spack-stack-new-joint/spack-stack-develop-20220215/envs/macos-clang-13.0.0-all-20220315/install/clang/13.0.0/crtm-2.3.0-fbk26m6/include")
```
because there are Fortran include files (`*.mod`) that some applications use.

I am not entirely sure if it is needed, but it seems safer to me to have the non-default gcc header/include files in the include search path when using/linking against code compiled with that version of gcc. I had added it to Cheyenne and it didn't hurt while building the spack-stack. I haven't tried compiling any downstream application on Cheyenne yet since we made the toolchain changes, though (but neither on orion).

Reply:
Sure, that sounds fine.

Reply:
Added CPATH